sql/wsrep_utils.h

```diff
@@ -225,6 +225,34 @@ class critical {
 };
 #endif
 
+
+class WSREPState
+{
+  public:
+    // Resets all of the data to default values
+    void clear() { wsrep_schema_version.clear(); }
+
+    bool load_from(const char *dir, const char *filename);
+    bool save_to(const char *dir, const char *filename);
+
+    // Compare the server version with the wsrep version
```

Comment:
Can you change this to use multi-line comment as per MySQL standard
/*
*/

Reply:
Fixed